x-pack/plugins/actions/server/usage/actions_usage_collector.ts

```diff
@@ -0,0 +1,69 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import { UsageCollectionSetup } from 'src/plugins/usage_collection/server';
+import { get } from 'lodash';
+import { TaskManagerStartContract } from '../../../task_manager/server';
+import { ActionsUsage } from './types';
+
+export function createActionsUsageCollector(
+  usageCollection: UsageCollectionSetup,
+  taskManager: TaskManagerStartContract
+) {
+  return usageCollection.makeUsageCollector({
+    type: 'actions',
```

Comment:
so does `usageCollection` take `type: actions` and then look for `actions-telemetry` types in the SO?  I'm not seeing any other references to `actions-telemetry` in the code, other than in the SO mappings at the top, so guessing this must be how it's done.